stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Can you remove stack.yaml? I think it's better to avoid having stack.yaml in a library repository, since it easily gets outdated and makes it harder for devs to test in multiple GHC versions.

Reply:
Oh, I stopped using Scotty years ago. You can feel free to close this PR or use any parts that are still useful.